src/components/inputs/reactQueryBuilder/CustomReactQueryBuilder.tsx

```diff
@@ -75,6 +82,10 @@ export function CustomReactQueryBuilder(props: Readonly<CustomReactQueryBuilderP
         }));
     }, [intl]);
 
+    // Drag handle title
+    const customDragHandle = { label: defaultTranslations.dragHandle.label, title: '' };
+    const customTranslations = { ...defaultTranslations, dragHandle: customDragHandle };
+
```

Comment:
We can use useMemo to override the whole defaultTranslations with empty title for components.
In the future, if we need translation, only replace empties by useInlt